app/views/lead_providers/guidance/ecf_usage.html.erb

```diff
@@ -46,19 +48,135 @@
 
 <p class="govuk-body-m">See <a href="/lead-providers/guidance/reference#get-api-v1-participants" class="govuk-link">retrieve multiple participants</a> endpoint.</p>
 
-<h2 id="declaring-that-an-ecf-participant-has-started-their-induction" class="govuk-heading-l">Declaring that an ECF participant has started their induction</h2>
+<h2 id="make-a-declaration" class="govuk-heading-l">Making a declaration for an ECF participant</h2>
+
+<h3 class="govuk-heading-m">
+  What is a declaration?
+</h3>
+<p class="govuk-body-m">
+  A declaration is a message in JSON format sent to the participant-declarations API.
+</p>
+
+<p class="govuk-body-m">
+  This will allow providers’ own software systems to integrate with our DfE service - these declarations will not need to be made manually by provider data teams.
+</p>
+
+<p class="govuk-body-m">
+A participant on the ECF programme is defined as either an Early Careers Teacher (ECT) or a mentor.
+</p>
+
+<p class="govuk-body-m">
+For a participant on a course, a declaration is the formal communication from a lead provider to the department that a training event for a specific participant has happened and can be evidenced on the providers’ platform.
+</p>
+
+<p class="govuk-body-m">
+  For example, participant Joe Bloggs who started their induction/course on a lead provider platform requires the lead provider to submit a <code>started</code> declaration for Joe Bloggs to the department.
+</p>
+
+
+<h3 class="govuk-heading-m">
+Declaration submission?
+</h3>
+
+<p class="govuk-body-m">
+Declarations provide contractual MI reporting requirements for lead providers.
+</p>
+
+<p class="govuk-body-m">
+They are the sole means for triggering a milestone payment to a lead provider for a participant.
+</p>
+
+<p class="govuk-body-m">
+They are submitted in line with the contractual milestones set-out by the DfE.
+</p>
+
+<p class="govuk-body-m">
+The participant-declarations API will accept the declarations required to trigger payments.
+</p>
+
+<p class="govuk-body-m">
+The validations process means declarations make sense in relation to the relevant milestone dates and the declarations previously submitted for the participant (available to test link maybe)
```

Comment:
One: let's use end lines, this diff gets hard to read on github.

Two:
>  (available to test link maybe)

ummmm... Is that the right copy?

Reply:
We probably need another round with the content people.